src/sentry/integrations/github/integration.py

```diff
@@ -27,10 +29,17 @@
 )
 
 
+class GitHubIntegration(Integration):
+
+    def get_client(self, integration):
+        return GitHubAppsClient(external_id=integration.external_id)
```

Comment:
you can change this to:
```python
def get_client(self):
    return GitHubAppsClient(external_id=self.model.external_id)
```
because the `Integration` class is instantiated with the integration model: https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/base.py#L157